### ISSUE TITLE
Implement Case CRUD operations

### ISSUE THREAD

**Issue by lamara11** [amosproj/amos2024ss01-xcelerator-demo-app#68]
## User story

1. As a facility manager
2. I want to be able to CRUD Items in the DB
3. So that I can perform orders/ list views 

## Acceptance criteria

- [x] An operation for creating cases is implemented
- [x] An operation for reading existing cases is implemented
- [x] An operation for updating cases is implemented
- [x] An operation for deleting existing cases is implemented

## Definition of done (DoD)

- [x] Added only after week 5
- [x] The same for all features
- [x] Here goes the project specific part

## DoD general criteria

- [x] Feature has been fully implemented
- [x] Feature has been merged into the mainline
- [x] All acceptance criteria were met
- [x] Product owner approved features
- [x] All tests are passing
- [x] Developers agreed to release

## Dependencies

- Item [#53](https://github.com/amosproj/amos2024ss01-xcelerator-demo-app/issues/53)

**Comment by sn0wcat** [comment]
Work Orders are now called Cases, we should name the stuff the same

https://developer.siemens.com/insights-hub/docs/apis/advanced-casemanagement/api-casemanagement-api.html

Here is an example Case:

```json
 {
      "handle": "AA-000",
      "dueDate": "2021-11-25",
      "notifyAssignee": false,
      "title": "test WOM 1",
      "type": "PLANNED",
      "status": "OPEN",
      "assignedTo": null,
      "description": "test wom",
      "source": "AHM",
      "priority": "LOW",
      "createdBy": "test@test.com",
      "createdDate": "2023-08-10T03:52:14.301Z",
      "eTag": "1702540787672",
      "modifiedBy": "test@test.com",
      "modifiedDate": "2023-08-10T03:52:14.301Z",
      "overdue": true
    }

```
